src/main/java/com/tenduke/events/api/model/object/LicenseConsumed.java

```diff
@@ -0,0 +1,21 @@
+package com.tenduke.events.api.model.object;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.tenduke.events.api.model.data.ErrorInfoFields;
+import com.tenduke.events.api.model.data.LicenseConsumptionFields;
+import com.tenduke.events.api.model.data.RequestFields;
+import com.tenduke.events.api.model.data.TimeFields;
+import com.tenduke.events.api.model.data.UserIdFields;
+
+/**
+ * Event occurring when the license is consumed.
+ * @author roope
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public interface LicenseConsumed extends
```

Comment:
As in the case of `LicenseChecked`, `LicenseConsumeFailed` can be removed and merged with `LicenseConsumed`.